Makefile.common

```diff
@@ -1,4 +1,4 @@
-REPOSITORY    := docker.elastic.co/beats-dev
+REPOSITORY    ?= docker.elastic.co/beats-dev
```

Comment:
This is the only change in the code to be able to set the repository where the docker images can be pushed, by default as used to be, but as we will be using the `staging` stage, then this will help to generate docker images on a PR basis which can be consumed for testing purposes.